tutorials/run-swift-vapor-on-google-cloud/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ibmcom/swift-ubuntu:latest
+LABEL Description="Docker image for Swift + Vapor on GAE Flex."
+
+# Expose default port for GAE
+EXPOSE 8080
+
+# Copy sources
+RUN mkdir /root/VaporGAE
+ADD main.swift /root/VaporGAE
+ADD Package.swift /root/VaporGAE
+
+# Build the app
+RUN cd /root/VaporGAE && swift build
```

Comment:
Awesome. https://github.com/GoogleCloudPlatform/appengine-custom-runtimes-samples/blob/master/swift/Dockerfile was a good example of this too, thanks for sending it over.